Detectors/TRD/simulation/src/Detector.cxx

```diff
@@ -42,7 +50,7 @@ Detector::Detector(const Detector& rhs)
   } else if (TRDCommonParam::Instance()->IsArgon()) {
     mWion = 27.21; // Ionization energy ArCO2 (82/18)
   } else {
-    LOG(FATAL) << "Wrong gas mixture";
+    LOG(FATAL) << "Wrong gas mixture" << FairLogger::endl;
```

Comment:
FairLogger::endl no longer necessary. There is an automatic end of line.

Reply:
Done.